### ISSUE TITLE
Add example how to use incremental indexing in custom indexer

### ISSUE THREAD

**Issue by ste101** [teaminmedias-pluswerk/ke_search_hooks#14]
It would be great if you could add an example how to use the incremental indexing in custom indexer.

**Comment by christianbltr** [comment]
Yes, that would be a good thing, I'm planning to add an example.

For now it may help if you have a look at the 'news' indexer of ke_search itself, it implements incremental indexing.
